packages/devtools_app/lib/src/shared/globals.dart

```diff
@@ -53,6 +54,8 @@ NotificationService get notificationService =>
 BreakpointManager get breakpointManager =>
     globals[BreakpointManager] as BreakpointManager;
 
+EvalService get evalService => globals[EvalService] as EvalService;
```

Comment:
I would remove this service or change where it is initialized. Seems a bit tightly coupled to the DebuggerController to have as a top level global unless there is no other way to conveniently achieve the same effect.